services/openuserjs/openuserjs-license.service.js

```diff
@@ -0,0 +1,38 @@
+import { renderLicenseBadge } from '../licenses.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSLicense extends BaseOpenUserJSService {
+  static category = 'license'
+  static route = { base: 'openuserjs', pattern: 'l/:username/:scriptname' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        username: 'NatoBoram',
+        scriptname: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderLicenseBadge({ licenses: ['GPL-3.0-or-later'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  transform(data) {
+    if (!('license' in data.UserScript)) {
+      // By the TOS, the script will be under an MIT License by default if not specified
+      // @see https://github.com/badges/shields/pull/8081#discussion_r898437399
```

Comment:
Perhaps this is something already well known to all users of the OpenUserJS platform regardless of how new they are to it, but I think it would be in our best interest to also mention this in our user facing documentation that's shown on the Shields.io page.

You can do this by adding a `documentation` key to the object in `examples` with whatever text you think will make the most sense, but if you choose to use the same wording as what's in this comment I'd suggest converting `TOS` to `terms of service` for explicitness